packages/components/src/DateTimePickers/MonthCalendar/MonthCalendarHeader/MonthCalendarHeader.component.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import theme from './MonthCalendarHeader.scss';
+
+function MonthCalendarHeader(props) {
```

Comment:
Calling the daypicker elements `MonthAnything` makes it hard to reason about or the find what we want based on the name.
It's because we have a month picker :/

Proposal:
* YearPicker
* MonthPicker
* DayPicker

In Day Picker we have 
* Calendar (that display a month calendar)
* CalendarWeekHeader
* CalendarDays
(or DayCalendar, DayCalendarHeader, DayCalendarBody)

Wrappers
* MonthYearWrapper
* DayTimeWrapper

Main component
* DateTimePicker